build/tasks/serve.js

```diff
@@ -10,10 +10,11 @@ gulp.task('serve', ['build'], function(done) {
   var bs = browserSync.create('Sample server');
 
   bs.init({
+    open: false,
```

Comment:
Oh oops. Forgot to take that out. It kept opening new browser instances on me... And it go annoying lol